### ISSUE TITLE
Squiz.PHP.LowercasePHPFunctions incorrectly reports on attribute names

### ISSUE THREAD

**Issue by bytestream** [squizlabs/PHP_CodeSniffer#3778]
**Describe the bug**
If an attribute name is also a PHP function name the LowercasePHPFunctionsSniff incorrectly throws an error.

**Code sample**
```php
<?php declare(strict_types=1);

#[Attribute]
class Putenv
{
    public function __construct(public string $name, public string $value)
    {}
}

class FooBar
{
    #[Putenv('FOO', 'foo')]
    public function foo(): void
    {}
}
```

**Custom ruleset**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset name="foo">
    <rule ref="Squiz.PHP.LowercasePHPFunctions"></rule>
</ruleset>
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```

FILE: /supportpal/test.php
--------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------------------------------------------
 12 | ERROR | [x] Calls to PHP native functions must be lowercase; expected "putenv" but found "Putenv"
    |       |     (Squiz.PHP.LowercasePHPFunctions.CallUppercase)
--------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 67ms; Memory: 4MB
```

**Expected behavior**
No error 

**Versions (please complete the following information):**
 - OS: Linux
 - PHP: 8.1
 - PHPCS: 3.7.2
 - Standard: Squiz

**Additional context**

**Comment by jrfnl** [comment]
@bytestream Thanks for reporting this. I've confirmed the issue and pulled PR #3779 to fix this. Testing appreciated.

**Comment by jrfnl** [comment]
FYI: the fix for this issue is included in today's [PHP_CodeSniffer 3.8.0 release](https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.8.0).

As per #3932, development on PHP_CodeSniffer will continue in the [PHPCSStandards/PHP_CodeSniffer](https://github.com/PHPCSStandards/PHP_CodeSniffer) repository. If you want to stay informed, you may want to start "watching" that repo (or watching releases from that repo).
